test/ng/snifferSpec.js

```diff
@@ -25,6 +25,25 @@ describe('$sniffer', function() {
     });
 
 
+    it('should be true if running inside nw.js', function() {
```

Comment:
I would move this test below `should be false on Chrome Packaged Apps` (or even incorporate it into that one).